### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by whois-marvin-42**
Thx

### CODE REVIEW

File: app/controllers/v1/lessons_controller.rb [hunk 1]
```diff
@@ -1,35 +1,99 @@
...
+      # params.permit(:id, lesson: [:title, :description])
+      @lesson.update!(update_params)
+      render json: @lesson
     end
 
     private
 
-    def lesson_params
+    def create_params
+      ActionController::Parameters.action_on_unpermitted_parameters = :raise
```
Comment (Showner) [inline app/controllers/v1/lessons_controller.rb]: useless?

File: app/controllers/v1/lessons_controller.rb [hunk 2]
```diff
@@ -1,35 +1,99 @@
...
+    # def deny_unpermitted_parameters
+    #   ActionController::Parameters.action_on_unpermitted_parameters = :raise
+    # end
+    def deny_unpermitted_parameters
+      ActionController::Parameters.action_on_unpermitted_parameters = :raise
+    end
+
+    # alias_method :update_params, :create_params
+    def update_params
+      ActionController::Parameters.action_on_unpermitted_parameters = :raise
```
Comment (Showner) [inline app/controllers/v1/lessons_controller.rb]: +1

File: spec/support/shared_examples/lesson_shared_examples.rb [hunk 3]
```diff
@@ -1,48 +1,60 @@
 RSpec.shared_examples "a valid lesson" do |a_status|
-  it "returns HTTP code #{Rack::Utils::SYMBOL_TO_STATUS_CODE[a_status]} (aka '#{a_status}')" do
-    expect(response).to have_http_status(a_status)
-  end
+  # it "returns HTTP code #{Rack::Utils::SYMBOL_TO_STATUS_CODE[a_status]} (aka '#{a_status}')" do
+  #   expect(response).to have_http_status(a_status)
+  # end
+  it {
+    subject
```
Comment (Showner) [inline spec/support/shared_examples/lesson_shared_examples.rb]: useless??

File: spec/controllers/v1/lessons_controller_spec.rb [hunk 4]
```diff
@@ -10,42 +69,298 @@
...
+    end
+
+    context "without description" do
+      let(:lesson_update_with) { attributes_for(:lesson).except!(:description) }
+      # it { is_expected.to have_http_status(:ok) }
+      include_examples "a valid lesson", :ok
+    end
+
+    context "with invalid description" do
+      it {
```
Comment (Showner) [inline spec/controllers/v1/lessons_controller_spec.rb]: 2 tests with wrong inputs vs 1 test in create with nil input

Comment (whois-marvin-42) [reply spec/controllers/v1/lessons_controller_spec.rb]: fixed in create

File: spec/support/shared_examples/lesson_shared_examples.rb (hunk 3 above)
Comment (whois-marvin-42) [reply spec/support/shared_examples/lesson_shared_examples.rb]: refactored

**Pull request merged**